scalafix/rules/src/main/scala/fix/Stable212Base.scala

```diff
@@ -185,6 +185,32 @@ trait Stable212Base extends CrossCompatibility { self: SemanticRule =>
     else Patch.empty
   }
 
+   def extractCollection(toCol: Tree): String = {
+     toCol match {
+       case Term.ApplyType(q"scala.Predef.fallbackStringCanBuildFrom", _) =>
+         "scala.collection.immutable.IndexedSeq"
+       case Term.ApplyType(Term.Select(coll,_), _) =>
+         coll.syntax
+       case Term.Apply(Term.ApplyType(Term.Select(coll, _), _), _) =>
+         coll.syntax
+       case Term.Select(coll,_) =>
+         coll.syntax
+       case _ => {
+         throw new Exception(
+           s"""|cannot extract collection from .to
+               |
+               |---------------------------------------------
+               |syntax:
+               |${toCol.syntax}
+               |
+               |---------------------------------------------
+               |structure:
+               |${toCol.structure}""".stripMargin
```

Comment:
Should we standardize the layout of error messages?

Reply:
This is from the breakout PR, I'm just going to reuse this function.